umm-spec-lib/src/cmr/umm_spec/migration/collection_progress_migration.clj

```diff
@@ -0,0 +1,28 @@
+(ns cmr.umm-spec.migration.collection-progress-migration
+  "Contains helper functions for migrating between version v1.9 and v1.10 of UMM CollectionProgress")
+
+(def mapping-up
+  {"COMPLETE" "COMPLETE"
+   "IN WORK"  "ACTIVE"
+   "PLANNED" "PLANNED"
+   "NOT APPLICABLE" "NOT APPLICABLE"
+   "NOT PROVIDED" "NOT PROVIDED"})
+
+(def mapping-down 
+  {"COMPLETE" "COMPLETE"
+   "ACTIVE"  "IN WORK"
+   "PLANNED" "PLANNED"
+   "NOT APPLICABLE" "NOT APPLICABLE"
+   "NOT PROVIDED" "NOT PROVIDED"} )
+
+(defn migrate-up
+  "Migrate CollectionProgress from string to enum."
+  [c]
+  (assoc c :CollectionProgress
+           (get mapping-up (:CollectionProgress c) "NOT PROVIDED")))
```

Comment:
I did. This file was actually not supposed to be here. because it's not used and outdated. I will remove it. and have the new updated one in part3.